src/test/java/ua/lviv/iot/graph/GraphTest.java

```diff
@@ -0,0 +1,60 @@
+package ua.lviv.iot.graph;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class GraphTest {
+	private Graph<String> graph;
+
+	public GraphTest() {
+		graph = new Graph<>();
```

Comment:
So... This graph var is used in several tests – what do you think happens between the tests? The var doesn't change the its value, obviously. What if I ask you to add a test that will add yet another Vertex to the graph and you use the same variable – your `assertEquals(graph.numberOfVertices(), 8);` will fail, since it unexpectedly have more vertexes.

You need to read about `org.junit.Test`. There're things like @Before @After @BeforeClass, etc